mutablesecurity/solutions/implementations/duplicati/meta.yaml

```diff
@@ -0,0 +1,8 @@
+full_name: Duplicati 2.0
+description: Duplicati is a backup client that securely stores encrypted, incremental, compressed backups on local storage, cloud storage services and remote file servers. 
+references:
+  - https://www.duplicati.com/
+  - https://github.com/duplicati/duplicati
+maturity: DEV_ONLY
+categories:
+  - NONE
```

Comment:
And add here a `BACKUP` category. Don't forget to define it into `solution.py` in `base` package.

Reply:
done